build.gradle

```
@@ -9,10 +9,9 @@ plugins {
     id 'maven'
 }
 
-def buildNumber = System.getenv("TRAVIS_BRANCH") ?: "DEV-SNAPSHOT"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "")
```

you probably need a fallback if its null to what you had before?
I think this would work
```suggestion
def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "DEV-SNAPSHOT"
```